lndr-backend/src/Lndr/Db/PendingCredits.hs

```diff
@@ -63,3 +63,18 @@ creditRecordToPendingTuple creditRecord = ( creditor creditRecord
                                           , ucac creditRecord
                                           , settlementCurrency creditRecord
                                           )
+
+
+insertPayPalRequest :: PayPalRequest -> Connection -> IO Int
+insertPayPalRequest r@(PayPalRequest friend requestor sign) conn = do
+    fromIntegral <$> execute conn "INSERT INTO paypal_requests (friend, requestor) VALUES (?,?)" (friend, requestor)
+
+
+deletePayPalRequest :: PayPalRequest -> Connection -> IO Int
+deletePayPalRequest r@(PayPalRequest friend requestor sign) conn = do
+    fromIntegral <$> execute conn "DELETE FROM paypal_requests WHERE (friend = ? AND requestor = ?)" (friend, requestor)
+
+
+lookupPayPalRequestsByAddress :: Address -> Connection -> IO [PayPalRequestPair]
+lookupPayPalRequestsByAddress userAddr conn = do
+    query conn "SELECT friend, requestor, targets.nickname, requestors.nickname FROM paypal_requests LEFT JOIN nicknames requestors ON requestors.address = requestor LEFT JOIN nicknames targets ON targets.address = friend WHERE (friend = ? OR requestor = ?)" (userAddr, userAddr) :: IO [PayPalRequestPair]
```

Comment:
:memo: There's some inconsistent naming with PayPal requests; in some places it's a requestor/**friend** pair, and in other cases it's a requestor/**target** pair; there doesn't seem to be a [compelling] contextual reason for the difference, so for clarity, it would be a minor readability improvement to apply "friend" or "target" consistently wherever a PayPal request is involved.

Reply:
:memo: While not strictly necessary, in SQL statements involving multiple tables, it's generally considered good practice to include the name of the table in all column references. For example, here instead of `friend`, you could use `paypal_requests.friend`.

Reason being, there's often ambiguity and similarities in the names of columns across tables (`id` is probably the most obvious one), and without knowledge of the whole structure of each table being queried, it can be difficult for a developer reading the query to track down which columns are associated with which tables.